instructionAPI/h/Operand.h

```diff
@@ -60,45 +60,16 @@ namespace Dyninst
     {
     public:
         typedef boost::shared_ptr<Operand> Ptr;
-    Operand() noexcept : m_isRead(false), m_isWritten(false), m_isImplicit(false), m_isTruePredicate(false), m_isFalsePredicate(false) {}
       /// \brief Create an operand from a %Expression and flags describing whether the %ValueComputation
       /// is read, written or both.
       /// \param val Reference-counted pointer to the %Expression that will be contained in the %Operand being constructed
       /// \param read True if this operand is read
       /// \param written True if this operand is written
-      Operand(Expression::Ptr val, bool read, bool written) : 
-          op_value(val), m_isRead(read), m_isWritten(written), m_isImplicit(false), m_isTruePredicate(false), m_isFalsePredicate(false) {}
-      Operand(Expression::Ptr val, bool read, bool written, bool implicit) : 
-          op_value(val), m_isRead(read), m_isWritten(written), m_isImplicit(implicit), m_isTruePredicate(false), m_isFalsePredicate(false) {}
       // An instruction can be true predicated, false predicated, or not predicated at all
-      Operand(Expression::Ptr val, bool read, bool written, bool implicit,
-              bool trueP, bool falseP):
+      Operand(Expression::Ptr val = {}, bool read = false, bool written = false, bool implicit = false,
```

Comment:
I would suggest making this `explict`, but I'm not sure how much that would break user code.

Reply:
Done; unlikely to impact user code.